src/index.html

```diff
@@ -16,7 +17,12 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <!-- #endregion -->
+  <body class="page">
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
According to the BEM methodology, modifiers should be used with the block or element name, not as standalone classes. In this case, `stopwatch--speed-up` should be an additional class alongside `stopwatch`, e.g., `class="stopwatch stopwatch--speed-up"`.